packages/web3-eth/src/index.js

```diff
@@ -385,6 +385,11 @@ var Eth = function Eth() {
             params: 0,
             outputFormatter: formatter.outputBigNumberFormatter
         }),
+        new Method({
+            name: 'getFeeHistory',
+            call: 'eth_feeHistory',
+            params: 3,
```

Comment:
```suggestion
            params: 4,
```

see comment below

Reply:
Pretty sure 3 is correct, its just `blockCount`, `newestBlock` and `rewardPercentiles`, from looking at the other methods the callback function doesn't count as a param